shims/spark300/src/main/scala/com/nvidia/spark/rapids/shims/spark300/Spark300Shims.scala

```diff
@@ -147,8 +147,9 @@ class Spark300Shims extends SparkShims {
       GpuOverrides.exec[FileSourceScanExec](
         "Reading data from files, often from Hive tables",
         ExecChecks((TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.STRUCT + TypeSig.MAP +
-            TypeSig.ARRAY).nested(), TypeSig.all),
+            TypeSig.ARRAY + TypeSig.DECIMAL).nested(), TypeSig.all),
```

Comment:
What about other shims that are specifying scan overrides like Spark310Shims?

Reply:
Yes, I missed it. And it leaded to the CI failure.